### PR TITLE
Adding svg icons as template files into the different template pages.

### PR DESCRIPTION
**Pull request by EdwinGuzman**

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
Work in progress but making it a pull request so @ricardoom can work off latest changes.

**Comment by EdwinGuzman** [comment]
@seanredmond this also fixes #35 
@ricardoom @aslagle Waiting on a couple more svgs which Ricardo will add in a different branch. This also contains all the latest CSS updates from Ricardo so this PR is now ready. I'll update the footer when the other svgs are in place.

**Comment by aslagle** [comment]
This looks fine. For the footer, the SimplyE logo doesn't work on a dark background. If it has to be there it should get its own white box or something.

**Comment by EdwinGuzman** [comment]
Yeah, that was a temporary fix just to have something there for now.

File: bin/generate_landing_page [hunk 1]
```diff
@@ -383,14 +389,14 @@ class GenerateLandingPageScript:
           for book in all_books_description:
               with open(BOOK_DESC_TEMPLATE) as template_file:
                   template = template_file.read()
-                  out = template.replace("{{title}}", all_books_description[book]['title']).replace("{{book_desc_html}}", all_books_description[book]['book_desc_html']).replace("{{category_nav}}", "".join(category_nav))
+                  out = template.replace("{{title}}", all_books_description[book]['title']).replace("{{book_desc_html}}", all_books_description[book]['book_desc_html']).replace("{{category_nav}}", "".join(category_nav)).replace("{{logo}}", logo).replace("{{menuOpenBtn}}", menuOpenBtn).replace("{{xIconBtn}}", xIconBtn)
```
Comment (EdwinGuzman) [inline bin/generate_landing_page]: @aslagle is there a way I can break this line into multiple lines?

Comment (aslagle) [inline bin/generate_landing_page]: yeah, 
```
out = template.replace("{{title}}", all_books_description[book]['title'])
out = out.replace("{{book_desc_html}}", all_books_description[book]['book_desc_html'])
out = out.replace...
```

Comment (EdwinGuzman) [inline bin/generate_landing_page]: Ah. I was hoping I could do that without declaring `out = ...` again. Thanks.

Comment (aslagle) [inline bin/generate_landing_page]: Well you can do line breaks in python by ending the line with `\`. I don't like to do that though.

Comment (aslagle) [inline bin/generate_landing_page]: you could also do
```
out = template.replace(
    "{{title}}", all_books_description[book]['title']
).replace(
    "{{book_desc_html}}", all_books_description[book]['book_desc_html']
).replace(
....
````

**Pull request merged**